src/librustdoc/html/static/search.js

```diff
@@ -106,7 +106,7 @@ function levenshtein(s1, s2) {
 window.initSearch = function(rawSearchIndex) {
     var MAX_LEV_DISTANCE = 3;
     var MAX_RESULTS = 200;
-    var GENERICS_DATA = 1;
+    var GENERICS_DATA = 2;
```

Comment:
Why did you change this?

Reply:
Because field 1 has the TypeKind in it. That’s why it didn’t work.

Reply:
Arf, side-effect from not enough tests... Thanks!